### ISSUE TITLE
review from george stigler

### ISSUE THREAD

**Issue by vennilaaa** [chrisgnicholas/EEP153_Harriette_Chick_Project2#1]
I had a little bit of a difficult time reviewing your code because everything was in a different file and when i ran the team file all I could see was this error no matter how much I scrolled. Would it be possible for you guys to put all your code in one notebook?
<img width="812" alt="image" src="https://github.com/chrisgnicholas/EEP153_Harriette_Chick_Project2/assets/94937932/55b03868-71a8-4c5a-bb38-293784b3a38b">

**Comment by chrisgnicholas** [comment]
yes, this is an annoying warning that happens inside the cufflinks plotting code of examine_price_changes.ipynb; putting everything back into a single notebook would not change this.  
But even though it is annoying, it shouldn't prevent you from running the team notebook, and the fact you got there means your runtime environment is finding all the dependencies.

Did you encounter any hard errors when you try to run all the way through?

We broke things apart for modularity and reuse (and test cases we didn't get around to writing  :-/   so our team notebook focuses on our data and plots.  It was much easier to compare two different age groups by simply calling the same code twice, than cutting-and-pasting into a single, gigantic notebook.
